### PR TITLE
adding skill specific pages

### PR DESCRIPTION
**Pull request by orlando**
# Context

This PR adds all the specific skill pages (secondary pages). Content is not ready yet for each page, so I added a boolean variable to reflect this and to hide placeholder content

## Notes
We still need to complete:

* Create videos and replace each video in the corresponding page
* #22 
* #28
* #33 

Before we can deploy

## Media

screencapture-127-0-0-1-4000-hire-devops-specialists-1489530990114

### CODE REVIEW

**Comment by orlando** [comment]
@duranmla please take of the pending items, so we can deploy this site tomorrow

File: _includes/footer-scripts.html [hunk 1]
```diff
@@ -12,3 +12,28 @@
 <script src="{{ "/js/typed.min.js" | prepend: site.baseurl }}"></script>
 <script src="{{ "/js/main.js" | prepend: site.baseurl }}"></script>
 <script src="{{ "/js/jqwidont-compressed.js" | prepend: site.baseurl }}"></script>
+
+<!-- Start of Async Drift Code -->
```
Comment (duranmla) [inline _includes/footer-scripts.html:16]: @orlando why we need all of this here?

**Review by duranmla — COMMENTED** [review]
Just a question before give my LGTM

File: _includes/footer-scripts.html (hunk 1 above)
Comment (orlando) [reply _includes/footer-scripts.html:16]: @duranmla we are using https://www.drift.com/

Comment (duranmla) [reply _includes/footer-scripts.html:16]: Awesome next time let's put some documentation within the PR

**Review by duranmla — APPROVED** [review]
💯

**Pull request merged**